contents/news/tips-on-how-to-prepare-a-talk/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Tips on preparing a talk
+date: 2015-03-20
+template: article.jade
+category: news
+---
+
+So you decided that you want to visit the JSunconf this year. As you might know a unconference lives from the visitors and he Talks are prepared by the visitors and get choosen in the morning of each conference day by the crowd. This gives everyone the chance to prepare an awesome talk. This is where you all come in.
+
+If you are terrified to speak in public or you have no idea hwo to prepare a talk you should have a look at [speaking.io](http://speaking.io/)
+
+If you work in a special environment, or you started an open source project or you have worked with a special library. It would be great if you can share your expirience with the audience. 
```

Comment:
special library. It would --> special library, it would
expirience --> experience